README.md

```diff
@@ -421,6 +421,11 @@ Sends a message to the Game Coordinator requesting to create a lobby.  Listen fo
 * `[options]` - See paramter description in [#createPracticeLobby]
 * `[callback]` - optional callback, returns args: `err, response`.
 
+*Note: it is unknown what should be supplied to `tournament_game_id` (See
+[#414](https://github.com/Arcana/node-dota2/issues/414)), but ticketed lobbies can be created with
```

Comment:
I prefer not suggesting people use createPracticeLobby for creating tournament games, as this would undermine the existence of the convenience method. I prefer we rather rend the `tournament_game_id` optional then